### PR TITLE
Update AR translations

### PR DESCRIPTION
**Pull request by xlmnxp**
My PR is a:

💅 Enhancement

Main update on the:

Plugin

### CODE REVIEW

File: packages/strapi-admin/admin/src/translations/ar.json [hunk 1]
```diff
@@ -156,7 +134,8 @@
   "ResetPasswordToken": "إعادة تعيين كلمة المرور",
   "Role": "قاعدة",
   "New entry": "إدخال جديد",
-  "request.error.model.unknow": "هذا النموذج غير موجود",
+  "request.error.model.unknown": "هذا النموذج غير موجود",
   "Users": "المستخدمين",
-  "Analytics": "إحصائيات"
-}
+  "Analytics": "إحصائيات",
+  "request.error.model.unknow": "هذا النموذج غير موجود"
```
Comment (kamalbennani) [inline packages/strapi-admin/admin/src/translations/ar.json]: Duplicated

File: packages/strapi-admin/admin/src/translations/ar.json [hunk 2]
```diff
@@ -156,7 +134,8 @@
...
+}
```
Comment (kamalbennani) [inline packages/strapi-admin/admin/src/translations/ar.json]: Add missing new line.

File: packages/strapi-admin/admin/src/translations/en.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{
+  {
```
Comment (kamalbennani) [inline packages/strapi-admin/admin/src/translations/en.json]: Remove trailing empty spaces

**Comment by xlmnxp** [comment]
Can you merging PR?

**Pull request merged**